docs/migrating-configs.md

```diff
@@ -58,7 +58,8 @@ storage:
   disks:
     - device: /dev/sda
       partitions:
-        - label: root
+        - number: 4
+          label: root
```

Comment:
For clarity, should we drop the `label` attribute?

Reply:
I like it as a safety-check so that it ensures that you're using the right now. WDYT?